addons/mail/static/src/core_ui/message.xml

```diff
@@ -75,11 +84,7 @@
                                     <i class="fa fa-calendar-o"/>
                                 </span>
                             </div>
-                            <div t-if="message.originThread === props.thread and message.notifications.length > 0" t-att-class="{ 'ms-2': !isAlignedRight }">
-                                <span class="o-mail-Message-notification cursor-pointer" t-att-class="message.failureNotifications.length > 0 ? 'text-danger' : text-600" role="button" tabindex="0" t-on-click="onClickNotification">
-                                    <i t-att-class="message.notifications[0].icon" role="img" aria-label="Delivery failure"/> <span t-if="message.notifications[0].label" t-out="message.notifications[0].label"/>
-                                </span>
-                            </div>
+                            <t t-if="!(!isAlignedRight and env.inChatWindow)" t-call="mail.MessageViewNotification"/>
```

Comment:
`env.inChatWindow` is unnecessary, `isAlignedRight` is already taking `env.inChatWindow` into account.
Also `!!isAlignedRight === isAlignedRight`